### PR TITLE
remove unused imports

### PR DESCRIPTION
**Pull request by nickpell**
The war continues.
I wonder what we could do to better catch these during PR review...

@calellowitz

### CODE REVIEW

**Comment by calellowitz** [comment]
can dimagimon not be set up to catch these? otherwise 👍 on ✅

File: custom/bihar/models.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import datetime
-from corehq.apps.change_feed import topics
-from custom.bihar import getters, BIHAR_DOMAINS
```
Comment (calellowitz) [inline custom/bihar/models.py]: looks like `BIHAR_DOMAINS` was required

**Comment by calellowitz** [comment]
test failure seems related
```
File "/mnt/commcare-hq-ro/custom/bihar/models.py", line 45, in CareBiharFluff

    domains = BIHAR_DOMAINS

NameError: name 'BIHAR_DOMAINS' is not defined
```

File: custom/bihar/models.py (hunk 1 above)
Comment (nickpell) [inline custom/bihar/models.py]: super weird, pycharm displays the import as unused when it's clearly not.

**Comment by nickpell** [comment]
✅

**Comment by nickpell** [comment]
> can dimagimon not be set up to catch these?

It used to be.  Not sure if that changed or became less obvious or if people just ignore it.

**Comment by emord** [comment]
They are caught, but only if you've changed that line in the PR. So if you remove the last reference to an import, but don't change the import's line it won't be caught by dimagimon

**Comment by calellowitz** [comment]
dimagimon also only catches things in changed code, so i guess it wouldnt be effective for the primary reason these exist, which is that the import used to be needed but no longer is. if the import statement isn't changed in a commit, dimagimon wont comment. dont know how easy it would be to change that behavior for only imports

**Comment by calellowitz** [comment]
@emord I see we left the same comment at the same time

**Pull request merged**